Edit: Getting started (#99)
FalixNodes-Software/help-center: _posts/minecraft/general/2022-01-29-getting-started.md

<div style="display: none" id="java" markdown=1>

Please read the first part of this guide before proceeding. You can find it [here](https://help.falixnodes.net/falix/general/getting-started/).

## Setting Up The Server

1. Go to the [Game Panel](https://panel.falixnodes.net).
2. Select your server.
3. Click on the start button, which is located at the upper left corner of the Console tab.

> 4. You will be greeted with a message asking you to choose a server type. In this case Minecraft Java. Type in the number beside the option given, for example `1`.

Reviewer: ```suggestion
4. You will be greeted with a message asking you to choose a server type. In this example, Minecraft Java. Type in the number beside the option you want. For example, enter `1` to select option 1.
```

> 5. You will be asked to choose a Server Software, if you don't know which to pick read our [Server Software guide](https://help.falixnodes.net/minecraft/general/server-softwares/).

Reviewer: ```suggestion
5. You will be asked to choose a Server Software, if you don't know which to pick, read our [Server Software guide](https://help.falixnodes.net/minecraft/general/server-softwares/).
```

> 6. You will be prompted to choose a Minecraft version, for example `1.17.1`.

Reviewer: ```suggestion
6. You will be prompted to choose a Minecraft version. For example, `1.17.1`.
```

7. You will see a popup asking you to agree to the Minecraft EULA. Click on "I Accept".

> Note: It usually takes a while for the server to boot the first time it's setup.

### Manually Setting A Java Version

1. Visit the [Game Panel](https://panel.falixnodes.net).

> 2. Find and click on your server, make sure it's off.

Reviewer: ```suggestion
2. Find and click on your server, make sure it's offline.
```

3. Go to the "STARTUP" tab.

> 4. Under "JAVA VERSION", insert in a Java version:

Reviewer: ```suggestion
4. Under "JAVA VERSION", insert a Java version:
```

    - if your server is **1.12** or below then type `adopt@1.8.0-275`.
    - if your server is **1.13** 1.16 then type `adopt@1.11.0-9`.
    - if your server is **1.17** then type `adopt@1.16.0-1`.

>     - if your server is **1.18** then type `1.17.0-custom=tgz+https://pterodactyl-api.catactyl.xyz/static-files/OpenJDK17U-jdk_x64_linux_hotspot_17.0.1_12.tar.gz`.

Reviewer: Have you tested this works and is a 'permanent' solution?

Author: I got all the info there from exo, I will double check tomorrow morning.

Author: I have double checked everything, it works perfectly fine.

5. You will be asked to choose a Server Software, if you don't know which to pick, read our [Server Software guide](https://help.falixnodes.net/minecraft/general/server-softwares/).
6. You will be prompted to choose a Minecraft version. For example, `1.18.1`.
7. You will see a popup asking you to agree to the Minecraft EULA. Click on "I Accept".

> Note: It usually takes a while for the server to boot the first time it's setup.

### Manually Setting A Java Version

1. Visit the [Game Panel](https://panel.falixnodes.net).
2. Find and click on your server, make sure it's offline.
3. Go to the "STARTUP" tab.
4. Under "JAVA VERSION", insert a Java version:

    - if your server is **1.12** or below then type `adopt@1.8.0-275`.
    - if your server is **1.13** 1.16 then type `adopt@1.11.0-9`.
    - if your server is **1.17** then type `adopt@1.16.0-1`.
    - if your server is **1.18** then type `1.17.0-custom=tgz+https://pterodactyl-api.catactyl.xyz/static-files/OpenJDK17U-jdk_x64_linux_hotspot_17.0.1_12.tar.gz`.

5. Go back to the console and start the server.

> > If this does not work, manually delete the `.jabba` folder in the File manager then start your server.

Reviewer: ```suggestion
> If this does not work, manually delete the `.jabba` folder in the file manager, then start your server.
```
